Scripts/release.sh

```diff
@@ -0,0 +1,276 @@
+#!/bin/bash
+
+# This script guides you through all the steps needed to do a new release.
+# The script does the following:
+
+# 1. Reminder prompt to update the CHANGELOG.
+# 2. Reminder prompt to update the Build Settings with the proper version number for each module that requires a version bump.
+# 3. Gets the version numbers from the XCode build settings.
+# 4. Update podspec files with the correct version number.
+# 5. Commit and push the version bump changes to devel.
+# 6. Prompt to merge devel onto master via GitHub UI.
+# 7. git tag all the modules.
+# 8. git push all tags.
+# 9. Verify podspec files.
+# 10. Confirm if pod trunk session is open.
+# 11. pod trunk push all the podspecs.
+
+
+# Change to the project root folder
+(cd ..;
```

Comment:
yes, all the way at the end of the script. i need the parenthesis to create a subshell for the directory change.